src/validation.cpp

```diff
@@ -4198,7 +4208,10 @@ void UnloadBlockIndex()
     mempool.clear();
     mapBlocksUnlinked.clear();
     vinfoBlockFile.clear();
-    nLastBlockFile = 0;
+    {
```

Comment:
Why the extra scope here? Its during init/deinit so everything is single-threaded anyway.

Reply:
Fixed!